src/Visualization/Python/Plot.py

```diff
@@ -100,6 +100,7 @@ def command(output, **kwargs):
                         )
                 else:
                     plt.savefig(output)
+                    plt.close()
```

Comment:
For my information, did you encounter an issue with this?

Reply:
Yeah when I was running the unit test on my machine, even when I had the `-o` option specified, it would still open the plot interactively causing a timeout. This stopped that from happening.